swarm/content.md

```diff
@@ -0,0 +1,107 @@
+# Swarm: a Docker-native clustering system
+
+`swarm` is a simple tool which controls a cluster of Docker hosts and exposes it
+as a single "virtual" host.
+
+`swarm` uses the standard Docker API as its frontend, which means any tool which
+speaks Docker can control swarm transparently: dokku, fig, krane, flynn, deis,
+docker-ui, shipyard, drone.io, Jenkins... and of course the Docker client itself.
+
+Like the other Docker projects, `swarm` follows the "batteries included but removable"
+principle. It ships with a simple scheduling backend out of the box, and as initial
+development settles, an API will develop to enable pluggable backends. The goal is
+to provide a smooth out-of-box experience for simple use cases, and allow swapping
+in more powerful backends, like `Mesos`, for large scale production deployments.
+
+## Installation
+
+###1 - Download and install the current source code.
+Ensure you have golang and git client installed (e.g. `apt-get install golang git` on Ubuntu).
+You may need to set `$GOPATH`, e.g `mkdir ~/gocode; export GOPATH=~/gocode`.
+
+The install `swarm` binary to your `$GOPATH` directory.
+
+```sh
+go get -u github.com/docker/swarm
+```
+
+###2 - Nodes setup
+The only requirement for Swarm nodes is to run a regular Docker daemon (version
+`1.4.0` and later).
+
+In order for Swarm to be able to communicate with its nodes, they must bind on a
+network interface. This can be achieved by starting Docker with the `-H` flag
+(e.g. `-H tcp://0.0.0.0:2375`).
+
+# Example usage
+
+```bash
+# create a cluster
+$ swarm create
```

Comment:
Yes, sorry about that - please hold off the review, I have another PR (https://github.com/docker/swarm/pull/367) on our main `README.md` waiting for review.

I'll update `content.md` with those changes as soon as they are in.